tests/test.cli.js

```diff
@@ -62,13 +46,6 @@ describe('Basic CLI tests', function() {
       'Not enough non-option arguments'));
   });
 
-  it('should show error if incorrect type', () => {
```

Comment:
Yeah initially I was just going to comment out the args and skip the tests. But that is pretty clunky. It seemed if we could use history to bring back the arg it might as well have the test there too. 

Less clutter that way.